client/packages/common/src/ui/components/buttons/standard/SplitButton.tsx

```diff
@@ -34,76 +35,79 @@ export const SplitButton = <T,>({
   isDisabled = false,
   selectedOption,
   onSelectOption,
+  label,
 }: SplitButtonProps<T>) => {
   const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
   const buttonLabel = selectedOption.label;
   const open = !!anchorEl;
 
   return (
-    <ButtonGroup color={color} variant="outlined" aria-label={ariaLabel}>
-      <ButtonWithIcon
-        color={color}
-        disabled={isDisabled}
-        sx={{
-          borderRadius: 0,
-          borderStartStartRadius: '24px',
-          borderEndStartRadius: '24px',
-        }}
-        onClick={() => {
-          onClick(selectedOption);
-        }}
-        label={buttonLabel}
-        Icon={Icon}
-      />
+    <Tooltip title={label}>
```

Comment:
Tried adding the tooltip in StatusChangeButton but it wouldn't show up.. so added directly to here.

The rest of the file is just reformatting..